hypothesis-python/tests/cover/test_random_module.py

```diff
@@ -103,3 +103,12 @@ def inner(x):
 
     entropy.RANDOMS_TO_MANAGE.remove(r)
     assert r not in entropy.RANDOMS_TO_MANAGE
+
+
+@given(st.random_module())
+def test_will_actually_use_the_random_seed(rnd):
+    a = random.randint(0, 100)
+    b = random.randint(0, 100)
+    random.seed(rnd.seed)
+    assert a == random.randint(0, 100)
+    assert b == random.randint(0, 100)
```

Comment:
I couldn't figure out how to model the bug as a test case in my issue. That's a slick way to put it 👍